### PR TITLE
Bump the version to 1.3.2

### PR DESCRIPTION
**Pull request by gaiksaya**
Signed-off-by: Sayali Gaikawad <gaiksaya@amazon.com>

### Description
Prepare for next patch release
 
### Issues Resolved
Part of post release activity https://github.com/opensearch-project/opensearch-build/issues/1805
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: server/src/main/java/org/opensearch/Version.java [hunk 1]
```diff
@@ -82,7 +82,8 @@
     public static final Version V_1_2_5 = new Version(1020599, org.apache.lucene.util.Version.LUCENE_8_10_1);
     public static final Version V_1_3_0 = new Version(1030099, org.apache.lucene.util.Version.LUCENE_8_10_1);
     public static final Version V_1_3_1 = new Version(1030199, org.apache.lucene.util.Version.LUCENE_8_10_1);
-    public static final Version CURRENT = V_1_3_1;
+    public static final Version V_1_3_2 = new Version(1030199, org.apache.lucene.util.Version.LUCENE_8_10_1);
```
Comment (zelinh) [inline server/src/main/java/org/opensearch/Version.java]: This is supposed to be `new Version(1030299,...`?

Comment (gaiksaya) [reply server/src/main/java/org/opensearch/Version.java]: Changed it. Thanks!

**Review by tlfeng — APPROVED** [review]
The changes are same with https://github.com/opensearch-project/OpenSearch/pull/2672/commits/ebbf55520c88b437c5f261f95f86309c56e89c30

**Comment by tlfeng** [comment]
> That is great. I did not realize that. Looks like DCO needs to be fixed for that issue.

😄 I said the changes are the same is to prove your changes are correct.
(I guess creating this PR just because it's a trouble to fix the DCO. 😁)

**Pull request merged**